### ISSUE TITLE
Update 1.2.3 breaks adapter for me

### ISSUE THREAD

**Issue by dotcom84** [iobroker-community-adapters/ioBroker.worx#71]
Adapter was working fine before.
Since the update it's in a restart loop with the following error:

worx.0 | 2020-09-05 09:37:53.868 | error | at mower.status.then.status (/opt/iobroker/node_modules/iobroker.worx/main.js:122:36)
-- | -- | -- | --
worx.0 | 2020-09-05 09:37:53.868 | error | (10908) TypeError: Cannot read property 'sc' of undefined
worx.0 | 2020-09-05 09:37:53.867 | error | (10908) unhandled promise rejection: Cannot read property 'sc' of undefined
Unhandled | 2020-09-05 09:37:53.866 | error | promise rejection. This error originated either by throwing inside of  an async function without a catch block, or by rejecting a promise which  was not handled with .catch().

**Comment by MeisterTR** [comment]
please post a debug log to see what data  are recived

**Comment by nicoh88** [comment]
Same here.

ioBroker debug log:


```
worx.0 | 2020-11-14 20:05:47.185 | info | (17621) Terminated (NO_ERROR): Without reason
-- | -- | -- | --
worx.0 | 2020-11-14 20:05:47.184 | info | (17621) terminating
worx.0 | 2020-11-14 20:05:47.171 | info | (17621) cleaned everything up...
worx.0 | 2020-11-14 20:05:47.170 | error | (17621) TypeError: Cannot read property 'sc' of undefined at /opt/iobroker/node_modules/iobroker.worx/main.js:122:36
worx.0 | 2020-11-14 20:05:47.169 | error | (17621) unhandled promise rejection: Cannot read property 'sc' of undefined
worx.0 | 2020-11-14 20:05:47.169 | error | Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
worx.0 | 2020-11-14 20:05:47.012 | debug | (17621) Worxcloud MQTT subscribe to PRM100/98D8633168A4/commandOut
worx.0 | 2020-11-14 20:05:47.011 | debug | (17621) Worxcloud MQTT connected to: a1optpg91s0ydf-ats.iot.eu-west-1.amazonaws.com
worx.0 | 2020-11-14 20:05:46.702 | debug | (17621) 2 Recieve User Devices: [{"id":328553,"product_id":49,"user_id":193341,"serial_number":"20193019840102157153","mac_address":"98D8633168A4","name":"WR141E","locked":true,"firmware_version":3.17
worx.0 | 2020-11-14 20:05:46.347 | debug | (17621) 1 Recieve User Certificate
worx.0 | 2020-11-14 20:05:45.974 | debug | (17621) 0 Recieve MqttServer Endpoint: a1optpg91s0ydf-ats.iot.eu-west-1.amazonaws.com
worx.0 | 2020-11-14 20:05:45.664 | silly | (17621) States user redis pmessage worx.0.*/worx.0.info.connection:{"val":true,"ack":true,"ts":1605380745663,"q":0,"from":"system.adapter.worx.0","user":"system.user.admin","lc":1605380745663}
worx.0 | 2020-11-14 20:05:45.662 | debug | (17621) Sucess connect to Worx Server!
worx.0 | 2020-11-14 20:05:45.661 | debug | (17621) Connect to worx ok : {"token_type":"Bearer","expires_in":31536000,"access_token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJhdWQiOiIxIiwianRpIjoiMTI3ODc3MDMwOGFiZWQ4MTVkY2FkNjhlOWQyMjFhODcxMjQ1N
worx.0 | 2020-11-14 20:05:44.021 | silly | (17621) States system redis pmessage system.adapter.worx.0.logLevel/system.adapter.worx.0.logLevel:{"val":"silly","ack":true,"ts":1605380743991,"q":0,"from":"system.adapter.worx.0","lc":1605380710357}
worx.0 | 2020-11-14 20:05:43.999 | silly | (17621) States user redis pmessage worx.0.*/worx.0.info.connection:{"val":false,"ack":true,"ts":1605380743992,"q":0,"from":"system.adapter.worx.0","user":"system.user.admin","lc":1605380743992}
worx.0 | 2020-11-14 20:05:43.973 | debug | (17621) Connect to worx....
worx.0 | 2020-11-14 20:05:43.972 | info | (17621) worx is selected
worx.0 | 2020-11-14 20:05:43.961 | info | (17621) starting. Version 1.2.3 in /opt/iobroker/node_modules/iobroker.worx, node: v12.19.0, js-controller: 3.1.6
host.frida | 2020-11-14 20:05:42.844 | info | instance system.adapter.worx.0 started with pid 17621
```

**Comment by MeisterTR** [comment]
fixed in 1.2.4
